endpoints-framework-tools/src/main/java/com/google/api/server/spi/tools/CloudClientLibGenerator.java

```diff
@@ -61,7 +61,7 @@ InputStream postRequest(String url, String boundary, String content) throws IOEx
     HttpRequestFactory requestFactory = new NetHttpTransport().createRequestFactory();
     HttpRequest request = requestFactory.buildPostRequest(new GenericUrl(url),
         ByteArrayContent.fromString("multipart/form-data; boundary=" + boundary, content));
-    request.setReadTimeout(60000);  // 60 seconds is the max App Engine request time
+    request.setReadTimeout(600000);  // Larger discovery documents require more than 20 sec to process
```

Comment:
This is surprising. More than a minute seems like something has gone wrong and the request should time out. None of our docs should be nearly large enough to go over a minute.

Reply:
I think it's just a combination of a larger discovery doc and a smaller compute core. We gradually hit the limit once in a while, with increasing frequency as our discovery doc grew. We currently have 295 endpoints currently.

Reply:
@elharo We've since reduced our discovery doc, but still run 50 sec and hit the limit once in a while with the upstream version. Are there any internal things you'd need to check on why this takes so long? From our perspective, increasing the timeout should be fine. Anyway, in case increasing the timeout is not viable upstream, the comment `60 seconds is the max App Engine request time` is not true anymore, the max App Engine request time is now 10 min.